README.md

```diff
@@ -215,6 +215,11 @@ In this case you can disable internal webpack:
  ],
 ```
 
+Also you should disable `webpack` option when you use Size Limit for tracking 
```

Comment:
1. `Also,` could be removed.
2. “disable `webpack` option” → “set `webpack: false`”

Reply:
Fixed.